5_ModelsContinued.md

```diff
@@ -119,12 +119,12 @@ Add the following to order_form_component.js:
     });
 
 ##Saving and updating a model
-Let's look at a few items in the code above. Notice that we're creating a new instance of a model (MenuOrderModel) in the createOrder function. Unlike data access methods, which are called statically off of the prototype, the save, update, and delete methods are called off of a specific instance of a model. So, if we want to create a new order, we will need to work with an instance of the MenuOrderModel.
+Let's look at a few items in the code above. Notice that we're creating a new instance of a model (MenuOrderModel) in the createOrder function. Unlike data access methods, which are called statically off of the prototype, the `save`, `update`, and `delete methods are called off of a specific instance of a model. So, if we want to create a new order, we will need to work with an instance of the MenuOrderModel.
 
-We assign the value of this.attr('details') to the MenuOrderModel's delivery property. If you recall, we bound the values of the name, address, and telephone number fields to the "delivery" object in the order_form.stache view template. Now, all we need to do to get the values of those fields is reference them off of the View Model's delivery property.
+We assign the value of this.attr('order') to the MenuOrderModel's delivery property. If you recall, we bound the values of the name, address, and telephone number fields to the "delivery" object in the order_form.stache view template. Now, all we need to do to get the values of those fields is reference them off of the View Model's delivery property.
```

Comment:
This seemed like a typo: `this.attr('details')`. I guessed that it was meant to be an "order". Didn't really look into it.